src/pages/user/UserOrganizations.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { User } from '../../types/user.types';
+import ButtonModal from './../../components/ButtonModal';
+
+interface UserOrganizationsProps {
+  user: User;
+}
+
+const UserOrganizations: React.FC<UserOrganizationsProps> = ({ user }) => {
+  const { t } = useTranslation('profile');
+
+  return (
+    <div>
```

Comment:
dobbelt med divs her? kanskje en div kan styles istedenfor? eller ikke ferdig?

Reply:
Ikke ferdig ennå, er for neste task